modules/dsp/NE10_fir.neon.s

```diff
@@ -532,10 +541,19 @@ dInp2_1          .dn   D31.F32
                     LDR         pState,[pStateStruct],#4
 
                     @//outBlockSize = blockSize / S->M
+#ifdef __PIC__
+                    @/* position-independent access of LDR pMask,=ne10_divLookUpTable */
+                    LDR         pTemp,.L_PIC1_GOT_OFFSET
+                    LDR         pMask,.L_GOT_ne10_divLookUpTable
+.L_PIC1:
+                    ADD         pTemp,pTemp,pc
+                    LDR         pMask,[pTemp,pMask]
+#else
                     LDR         pTemp,=ne10_divLookUpTable
```

Comment:
I'm far from an expert on PIC, but I don't think that this line should be changed. [`LDR Rt, .label`](http://infocenter.arm.com/help/index.jsp?topic=/com.arm.doc.dui0041c/Babbfdih.html) is an instruction to load a `pc` relative value within a certain range into a register, while `LDR Rt, =.label` is a pseudo-instruction that loads an absolute address into a register via a literal pool. In this case, the behaviour of the the former is what we want as `pTemp` and `pMask` are supposed to contain the words at those labels.

To outline more explicitly what's going on here, the `ADD` instruction sums the relative offset between the GOT and the end of the final `LDR` instruction (i.e. `_GLOBAL_OFFSET_TABLE_-(.L_PICX+4)`) with the actual address after the final `LDR` instruction at execution (i.e. `pc`, which is the absolute address of `(.L_PICX+4)` as we're in Thumb mode) to get the absolute address of the GOT. This is then used in combination with the offset of the data (e.g. `ne10_qMaskTable32`) within the lookup table to perform the position-independent load.